src/crypto/CHIPCryptoPALOpenSSL.cpp

```diff
@@ -1665,5 +1666,73 @@ CHIP_ERROR ExtractPubkeyFromX509Cert(const ByteSpan & certificate, Crypto::P256P
     return err;
 }
 
+CHIP_ERROR ExtractAKIDFromX509Cert(const ByteSpan & certificate, MutableByteSpan & akid)
```

Comment:
Suggest having a SKID extraction as well.

Reply:
upcoming PR for CMS Certification Declaration will include it

Reply:
Wondering if the upcoming PR you mentioned might be conflicting with #9285. Asking so we have better tasks coordination

Reply:
we weren't aware of PR 9285. I see that that PR already includes a method to extract Subject Key ID from the CMS Envelope, therefore we won't need to publish our changes. In that case I see no need to add a method to ExtractSKIDFromX509Cert since there is no use case scenario to use that for now.